lib/index.js

```diff
@@ -61,6 +61,7 @@ internals.Client = class {
             Hoek.assert(internals.isNullOrUndefined(options.beforeRedirect) || typeof options.beforeRedirect === 'function', 'options.beforeRedirect must be a function');
             Hoek.assert(internals.isNullOrUndefined(options.redirected) || typeof options.redirected === 'function', 'options.redirected must be a function');
             Hoek.assert(options.gunzip === undefined || typeof options.gunzip === 'boolean' || options.gunzip === 'force', 'options.gunzip must be a boolean or "force"');
+            Hoek.assert(options.lookup === undefined || typeof options.lookup === 'function', 'options.lookup must be a function');
```

Comment:
I'm not sure we actually need to `assert()` this. We only check wreck specific options as it is.